README.md

```diff
@@ -1,18 +1,27 @@
 # Ultra Zoom
 
-A fast single image super-resolution (SISR) model for upscaling images without loss of detail. Ultra Zoom uses a two-stage "zoom in and enhance" strategy that uses a fast deterministic upscaling algorithm to zoom in and then enhances the image through a residual pathway that operates primarily in the low-resolution subspace of a deep neural network. As such, Ultra Zoom requires less resources than upscalers that predict every new pixel de novo - making it outstanding for real-time image processing.
+A fast single image super-resolution (SISR) model for upscaling images without loss of detail. Ultra Zoom uses a
+two-stage "zoom in and enhance" strategy that uses a fast deterministic upscaling algorithm to zoom in and then enhances
```

Comment:
Does your IDE not support line wrap?

Reply:
Nice, is that why it added line breaks? Is it the editor that does that or git iteself?

Reply:
@andrewdalpino Can you please provide your own .editorconfig file with your own codestyle so I can autochange everything before merging this PR? Again, by providing an .editorconfig file as a single "source of truth" for your codestyle anybody contributing to this repo will have EXACTLY same codestyle REGARDLESS their IDE or code editor. Try consider this please.